### PR TITLE
added tables for cache parameters

### PR DESCRIPTION
**Pull request by dvdksn**

### CODE REVIEW

File: docs/guides/cache/azblob.md [hunk 1]
```diff
@@ -27,20 +27,18 @@ $ docker buildx build . --push -t <registry>/<image> \
...
+
+| Parameter           | Option     | Value      | Default | Description                                        |
+| ------------------- | ---------- | ---------- | ------- | -------------------------------------------------- |
+| `name`              | Both       | string     |         | Required. The name of the cache image.             |
+| `account_url`       | Both       | string     |         | Base URL of the storage account.                   |
+| `secret_access_key` | Both       | string     |         | Blob storage account key, see [authentication][1]. |
+| `mode`              | `cache-to` | `min\|max` | `min`   | Cache layers to export, see [cache mode][2].       |
+
+[1]: #authentication
+[2]: ../#cache-mode
```
Comment (dvdksn) [inline docs/guides/cache/azblob.md:41]: This is a broken link when reading the markdown file, but I am pretty sure the other syntax (`./index.md#cache-mode`) breaks on the rendered docs site?

Comment (crazy-max) [reply docs/guides/cache/azblob.md:41]: or maybe there is an issue with reverse footnotes with Jekyll :thinking:

Comment (dvdksn) [reply docs/guides/cache/azblob.md:41]: I was misremembering that mixing file-based references with HTML IDs would somehow cause broken links... But I tested it now and it works with Jekyll. Might've been something from Hugo. I will fix this link.

**Comment by dvdksn** [comment]
@crazy-max what do you think of these tables for the cache params...

The 2nd column is a bit awkward but I have no better suggestion for how to show which parameters are for import/export/both

**Comment by crazy-max** [comment]
DCO sign-off looks missing: https://github.com/docker/buildx/pull/1332/checks?check_run_id=8625353776

**Comment by dvdksn** [comment]
> DCO sign-off looks missing: https://github.com/docker/buildx/pull/1332/checks?check_run_id=8625353776

yeah sorry, forgot! should be fixed now.

**Review by crazy-max — APPROVED** [review]
LGTM, PTAL @jedevc

File: docs/guides/cache/local.md [hunk 2]
```diff
@@ -26,30 +26,27 @@ $ docker buildx build . --push -t <registry>/<image> \
...
+| `src`               | `cache-from` | String                |         | Path of the local directory where cache gets imported from.          |
+| `digest`            | `cache-from` | String                |         | Digest of manifest to import, see [cache versioning][4].             |
+| `dest`              | `cache-to`   | String                |         | Path of the local directory where cache gets exported to.            |
+| `mode`              | `cache-to`   | `min\|max`            | `min`   | Cache layers to export, see [cache mode][1].                         |
+| `oci-mediatypes`    | `cache-to`   | `true\|false`         | `true`  | Use OCI media types in exported manifests, see [OCI media types][2]. |
+| `compression`       | `cache-to`   | `gzip\|estargz\|zstd` | `gzip`  | Compression type, see [cache compression][3].                        |
+| `compression-level` | `cache-to`   | `0..22`               |         | Compression level, see [cache compression][3].                       |
+| `force-compression` | `cache-to`   | `true\|false`         | `false` | Forcibly apply compression.                                          |
```
Comment (jedevc) [inline docs/guides/cache/local.md:41]: "Forcibly apply compression, see cache compression" (should have same style as the row above).

**Review by jedevc — CHANGES_REQUESTED** [review]
LGTM, just a couple minor nits, the table reads a lot easier to me :tada:

File: docs/guides/cache/azblob.md [hunk 3]
```diff
@@ -27,20 +27,18 @@ $ docker buildx build . --push -t <registry>/<image> \
...
-Parameters for `--cache-to`:
-
-- `mode`: specify cache layers to export (default: `min`), see
-  [cache mode](./index.md#cache-mode)
+The following table describes the available CSV parameters that you can pass to
+`--cache-to` and `--cache-from`.
+
+| Parameter           | Option     | Value      | Default | Description                                        |
+| ------------------- | ---------- | ---------- | ------- | -------------------------------------------------- |
+| `name`              | Both       | String     |         | Required. The name of the cache image.             |
```
[the remark this answers is not in the document]
Comment (dvdksn) [reply docs/guides/cache/azblob.md]: Agree, I found "both" very awkward so trying to list both instead

btw, trying to use the comma as the separator now... the escaped pipe that I used before doesnt seem to work w/ jekyll, and a raw pipe will mess up both autoformatting and linter apparently, even if it's in backticks.

File: docs/guides/cache/azblob.md [hunk 4]
```diff
@@ -27,20 +27,18 @@ $ docker buildx build . --push -t <registry>/<image> \
...
-  see [authentication](#authentication).
-
-Parameters for `--cache-to`:
-
-- `mode`: specify cache layers to export (default: `min`), see
-  [cache mode](./index.md#cache-mode)
+The following table describes the available CSV parameters that you can pass to
+`--cache-to` and `--cache-from`.
+
+| Parameter           | Option     | Value      | Default | Description                                        |
```
[the remark this answers is not in the document]
Comment (dvdksn) [reply docs/guides/cache/azblob.md]: +1

**Review by jedevc — APPROVED** [review]

**Pull request merged**